metals/src/main/scala/scala/meta/internal/builds/GradleDigest.scala

```diff
@@ -24,7 +24,7 @@ object GradleDigest extends Digestable {
   }
 
   def digestBuildSrc(path: AbsolutePath, digest: MessageDigest): Boolean = {
-    Files.walk(path.toNIO).iterator().asScala.forall { file =>
+    Files.walk(path.toNIO).allMatch { file =>
```

Comment:
Both `.walk` and `list` mention:
```
     * <p> The returned stream encapsulates one or more {@link DirectoryStream}s.
     * If timely disposal of file system resources is required, the
     * {@code try}-with-resources construct should be used to ensure that the
     * stream's {@link Stream#close close} method is invoked after the stream
     * operations are completed.  Operating on a closed stream will result in an
     * {@link java.lang.IllegalStateException}.
```

I am wondering if the issue is that we don't call `close()` on those methods. `digestBuildSrc` is not used very often, since that is used to define plugins in-build, so I suspect this is not the issue.

Reply:
Although, I might be completely wrong here.